src/test/lib/sql/sql_translator_test.cpp

```diff
@@ -3097,8 +3099,10 @@ TEST_F(SQLTranslatorTest, CastStatement) {
   EXPECT_THROW(sql_to_lqp_helper("SELECT CAST(a AS DATE) FROM int_string"), InvalidInputException);
   // We do not have a real Date data type, so we cannot pass this cast anywhere else.
   EXPECT_THROW(sql_to_lqp_helper("SELECT CAST(b AS DATE) FROM int_string"), InvalidInputException);
-  // This is valid SQL, but we do not have a DateTime data type
-  EXPECT_THROW(sql_to_lqp_helper("SELECT CAST('2000-01-01 00:00:00' AS DATETIME)"), InvalidInputException);
+  EXPECT_THROW(sql_to_lqp_helper("SELECT CAST('2000-01-01' AS DATETIME)"), InvalidInputException);
```

Comment:
Test for too short integers? Such as `10-01-01 00:00:00` or `'2020-01-01 0:0:17`?

Reply:
Currently, the latter should be acceptable (at least boost accepts it, we can change this if you like). Gonny add a test for the first case.